scss/sin-core/grid/_simple-grid-mixin.scss

```diff
@@ -1,4 +1,4 @@
-@use '../../../../../sin.config' as *;
+// @use '../../../../../sin.config' as *;
```

Comment:
I'm not sure where this comes from, or what it's used for -- but regardless, it doesn't exist inside my fork of the repo :)

Reply:
It's not a very delicate solution I know, but it works for me for now. This is a configuration file for a project. It keeps various maps containing things like breakpoints, colors etc. 

In some files it's not needed at all, and is probably there by mistake. Something that I will keep in mind to test and fix in the future